docs/website/docs/general-usage/schema.md

```diff
@@ -197,70 +159,56 @@ Now go ahead and try to add a new record where `id` is float number, you should
-3. Nested tables get a column with `parent_key` hint (named `_dlt_parent_id` by default) that refers to `row_key` of the `parent` table.
+1. All tables get a column with the `row_key` hint (named `_dlt_id` by default) to uniquely identify each row of data.
+2. Nested tables get the `parent` table hint with the name of the parent table. The root table does not have the `parent` hint defined.
+3. Nested tables get a column with the `parent_key` hint (named `_dlt_parent_id` by default) that refers to the `row_key` of the `parent` table.
 
-`parent` + `row_key` + `parent_key` form a **nested reference**: from nested table to `parent` table and are extensively used when loading data. Both `replace` and `merge` write dispositions
+`parent` + `row_key` + `parent_key` form a **nested reference**: from the nested table to the `parent` table and are extensively used when loading data. Both `replace` and `merge` write dispositions.
 
 `row_key` is created as follows:
-1. Random string on **root** tables, except for [`upsert`](incremental-loading.md#upsert-strategy) and
-[`scd2`](incremental-loading.md#scd2-strategy) merge strategies, where it is a deterministic hash of `primary_key` (or whole row, so called `content_hash`, if PK is not defined).
-2. A deterministic hash of `parent_key`, `parent` table name and position in the list (`_dlt_list_idx`)
-for **nested** tables.
+1. Random string on **root** tables, except for [`upsert`](incremental-loading.md#upsert-strategy) and [`scd2`](incremental-loading.md#scd2-strategy) merge strategies, where it is a deterministic hash of `primary_key` (or the whole row, so-called `content_hash`, if PK is not defined).
+2. A deterministic hash of `parent_key`, `parent` table name, and position in the list (`_dlt_list_idx`) for **nested** tables.
 
-You are able to bring your own `row_key` by adding `_dlt_id` column/field to your data (both root and nested). All data types with equal operator are supported.
+You are able to bring your own `row_key` by adding the `_dlt_id` column/field to your data (both root and nested). All data types with an equal operator are supported.
 
-`merge` write disposition requires additional nested reference that goes from **nested** to **root** table, skipping all parent tables in between. This reference is created by [adding a column with hint](incremental-loading.md#forcing-root-key-propagation) `root_key` (named `_dlt_root_id` by default) to nested tables.
+The `merge` write disposition requires an additional nested reference that goes from the **nested** to the **root** table, skipping all parent tables in between. This reference is created by [adding a column with the hint](incremental-loading.md#forcing-root-key-propagation) `root_key` (named `_dlt_root_id` by default) to nested tables.
 
 ### Table references
-You can annotate tables with table references. This feature is coming soon.
+You can annotate tables with table references. This feature is coming soon.## Schema settings
```

Comment:
this header moved down here, maybe write a regex to find these kind of occurences